examples/jsm/renderers/CSS3DRenderer.js

```diff
@@ -246,7 +247,9 @@ class CSS3DRenderer {
 					if ( object.rotation2D !== 0 ) _matrix.multiply( _matrix2.makeRotationZ( object.rotation2D ) );
 
 					_matrix.copyPosition( object.matrixWorld );
-					_matrix.scale( object.scale );
+					const objectWorldScale = new Vector3();
```

Comment:
Instantiating a new object every call is not going to be acceptable.

Create `_vector`,  `_quaternion` and `_scale` and reuse them. Follow the advice of @Mugen87 and use `decompose()`.

Reply:
Ok, I have put them at the top of the file and now there is decompose